save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -68,46 +68,30 @@ class VulnerabilityController(
         vulnerabilityService.getFilteredWithUserInfos(filters, authentication)
     }
 
-    @GetMapping("/by-name-and-status")
+    @GetMapping("/by-identifier")
```

Comment:
I named it as `cosv_id` in metadata. What do you which name is better?

Reply:
to my mind, identifier is a bit more clear, as I don't know the scheme of `cosv`, while `identifier` is more clear in case of vulnerabilities